pf/init.sls

```diff
@@ -1,11 +1,18 @@
 {% from 'pf/map.jinja' import pf with context %}
 
+load_kernel_module:
```

Comment:
Maybe we can prefix this by `{{ pf.kmodule }}` so it's more "unique" and we reduce the chances to collide with other state in a highstate.

Reply:
Changed to `pf_load_{{ pf.kmodule }}`